servers/rendering/rendering_device.h

```diff
@@ -1383,6 +1383,13 @@ class RenderingDevice : public Object {
 	};
 
 	Error _reflect_spirv(const Vector<ShaderStageSPIRVData> &p_spirv, SpirvReflectionData &r_reflection_data);
+
+#ifndef DISABLE_DEPRECATED
+	BitField<BarrierMask> _convert_barrier_mask_81356(BitField<BarrierMask> p_old_barrier);
+	void _draw_list_end_bind_compat_81356(BitField<BarrierMask> p_post_barrier);
+	void _compute_list_end_bind_compat_81356(BitField<BarrierMask> p_post_barrier);
+	void _barrier_bind_compat_81356(BitField<BarrierMask> p_from, BitField<BarrierMask> p_to);
+#endif
```

Comment:
Is there a reason these aren't declared next to `_bind_compatibility_methods()` and the pre-existing `_shader_create_from_bytecode_bind_compat_79606`? If not, it'd probably be good to group these over by those.

Reply:
Alternatively, you could move the pre-existing declarations down by your new ones? Although, I think it's probably fine either way

Reply:
Should be fine as is. We can always move them around for better code organization.